lib/src/interfaces.dart

```diff
@@ -42,36 +42,36 @@ class AnnotationLayerBuilderOptions {
     _pdfPage = pdfPage;
   }
 
-  bool get renderInteractiveForms => _jsInternal['renderInteractiveForms'] as bool;
-  set renderInteractiveForms(bool renderInteractiveForms) {
+  bool? get renderInteractiveForms => _jsInternal['renderInteractiveForms'] as bool?;
+  set renderInteractiveForms(bool? renderInteractiveForms) {
     _jsInternal['renderInteractiveForms'] = renderInteractiveForms;
   }
 }
 
 enum AnnotationLayerBuilderIntent { display }
 
 class AnnotationLayerBuilder {
-  JsObject _jsInternal;
+  JsObject? _jsInternal;
 
   AnnotationLayerBuilder(AnnotationLayerBuilderOptions options) {
-    _jsInternal = JsObject(PDFJS.versionSafePdfJsViewerContext['AnnotationLayerBuilder'] as JsFunction, [
+    _jsInternal = JsObject(PDFJS.versionSafePdfJsViewerContext!['AnnotationLayerBuilder'] as JsFunction, [
       options,
     ]);
   }
 
   AnnotationLayerBuilder._withJsInternal(this._jsInternal);
 
   void render(PageViewport viewport) {
-    _jsInternal.callMethod('render', [viewport._jsInternal]);
+    _jsInternal!.callMethod('render', [viewport._jsInternal]);
```

Comment:
`_jsInternal` is a type of `JSObject?` so It seems we cannot apply `!` on this line instead of can we use  `_jsInternal?.callMethod`